kolibri/core/errorreports/schemas.py

```diff
@@ -0,0 +1,70 @@
+context_frontend_schema = {
+    "type": "object",
+    "properties": {
+        "browser": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string", "optional": True},
+                "major": {"type": "string", "optional": True},
+                "minor": {"type": "string", "optional": True},
+                "patch": {"type": "string", "optional": True},
+            },
+        },
+        "component": {"type": "string", "optional": True},
+        "os": {
+            "type": "object",
+            "properties": {
+                "name": {"type": "string", "optional": True},
+                "major": {"type": "string", "optional": True},
+                "minor": {"type": "string", "optional": True},
+                "patch": {"type": "string", "optional": True},
```

Comment:
See https://github.com/learningequality/kolibri/pull/12382/files#r1675804546

Reply:
For ecample it would mean adding definitions

```json
....
"definitions": {
    "versionInfo": {
      "type": "object",
      "properties": {
        "name": {"type": "string", "optional": True},
        "major": {"type": "string", "optional": True},
        "minor": {"type": "string", "optional": True},
        "patch": {"type": "string", "optional": True},
      }
    }
  }
  ```
  
  and then use it as below;
  ```json
  "os":  {
      "$ref": "#/definitions/versionInfo",
  },
  ...
  "browser": {
      "$ref": "#/definitions/versionInfo",
  }
  ```